### PR TITLE
CAT-2926: Update documentation to include BE when talking about autom…

### PR DESCRIPTION
**Pull request by NammNguyen**
…atic registration

### CODE REVIEW

**Comment by NammNguyen** [comment]
Screenshot 2024-09-10 at 11 19 45
Hi Quinn, I would like to replace the image used on /catalog/ (currently: attachments/catalog/catalog-home.png) with the following image. How would I do that?

**Comment by quinntracy** [comment]
@NammNguyen I can replace it on my end

File: content/en/docs/catalog/get-started.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can use the Catalog with any deployment method, though there are differences
 
 | Deployment Method | How are Data Sources Registered? | 
 | --- | --- |
-| [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/) | Published OData services in a deployed app are [registered automatically](/catalog/register/register-data/#mendix-cloud) as data sources in the Catalog <br> [Register sources using the Registration API](/catalog/register/register-data/#register-services) |
+| [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/) | Published OData/REST services and Business Events servces in a deployed app are [registered automatically](/catalog/register/register-data/#mendix-cloud) as data sources in the Catalog <br> [Register sources using the Registration API](/catalog/register/register-data/#register-services) |
```
Comment (suyebmendix) [inline content/en/docs/catalog/get-started.md]: Service is misspelled here in `Business Events servces`.

Comment (quinntracy) [reply content/en/docs/catalog/get-started.md]: Fixed

**Comment by quinntracy** [comment]
Planning for 17 October merge

**Pull request merged**